string model_type
string model_name
string model_reference
---
